src/foam/u2/images/sdv-icon.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="18" height="16" viewBox="0 0 18 16">
+    <g fill="none" fill-rule="evenodd">
+        <path d="M-3-4h24v24H-3z"/>
```

Comment:
What does sdv stand for in the filename "sdv-icon", can we expand that acronym?

Reply:
Just removed this file instead, used tree view icon.